### ISSUE TITLE
Front facing Camera result very dim

### ISSUE THREAD

**Issue by jaydangar** [Fotoapparat/Fotoapparat#331]
@Diolor , @dmitry-zaitsev Hello there, first of of i want to give a big thumbs up for this great library that u guys are building. I am not criticizing, but front facing camera after setting exposure to highest level remains dim. I don't know if this is only in my camera or any other camera is facing the same. FYI, i am using Moto g5 plus and in built camera app can make picture really very bright. Please have a look at it as well. 

for proof i am attaching following photographs taken from front facing camera with your library as well as with default camera app.

**Comment by dmitry-zaitsev** [comment]
Hi @jaydangar , thanks for reporting this.

Could you please help us with the following:

- Show us how you are initializing `Fotoapparat` instance.
- Share an actual photo. It appears that you attached a screenshot, not the photo.

**Comment by jaydangar** [comment]
@dmitry-zaitsev , yes i can. this is how i am doing it. I have made the exposure level to highest still not able to get desired result.

`   fotoapparat = Fotoapparat
                .with(this)
                .into(cameraView)           // view which will draw the camera preview
                .previewScaleType(ScaleType.CenterCrop)  // we want the preview to fill the view
                .jpegQuality(JpegQualitySelectorsKt.manualJpegQuality(50))
                .photoResolution(ResolutionSelectorsKt.highestResolution())   // we want to have the biggest photo possible
                .lensPosition(LensPositionSelectorsKt.front())       // we want front camera
                .focusView(focusView)
                .logger(LoggersKt.loggers(            // (optional) we want to log camera events in 2 places at once
                        LoggersKt.logcat(),           // ... in logcat
                        LoggersKt.fileLogger(this)    // ... and to file
                )).focusMode(SelectorsKt.firstAvailable(  // (optional) use the first focus mode which is supported by device
                        FocusModeSelectorsKt.continuousFocusPicture(),
                        FocusModeSelectorsKt.autoFocus(),        // in case if continuous focus is not available on device, auto focus will be used
                        FocusModeSelectorsKt.fixed()             // if even auto focus is not available - fixed focus mode will be used
                ))
                **.exposureCompensation(highest())**
                .sensorSensitivity(highestSensorSensitivity())
                .cameraErrorCallback(e -> {
                    Log.d(TAG, "onError: CameraException -> " + e.toString());
                    intentTasks.passIntent(CameraActivity.this, ProfileActivity.class);
                })
                .previewFpsRange(highestFps())
                .build();
`
